src/background.js

```diff
@@ -16,11 +16,22 @@ function log(...args) {
   if (DEBUG) { console.log('SnoozeTabs (BE):', ...args); }  // eslint-disable-line no-console
 }
 
+var iconData;
```

Comment:
At the top level, they're the same!  😁  But adding stuff to the eslint config sounds like a good idea.